tests/e2e/restart_test.go

```diff
@@ -21,39 +21,68 @@ package e2e
 
 import (
 	. "gopkg.in/check.v1"
+	"strings"
+	"log"
 
 	"github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
 	"github.com/vmware/docker-volume-vsphere/tests/utils/inputparams"
 	"github.com/vmware/docker-volume-vsphere/tests/utils/verification"
 	"github.com/vmware/docker-volume-vsphere/tests/utils/misc"
+	adminconst "github.com/vmware/docker-volume-vsphere/tests/constants/admincli"
+	adminutil "github.com/vmware/docker-volume-vsphere/tests/utils/admincli"
 )
 
 type RestartTestData struct {
 	config        *inputparams.TestConfig
 	volumeName    string
-	containerName string
+	containerNameList []string
 }
 
 func (s *RestartTestData) SetUpSuite(c *C) {
 	s.config = inputparams.GetTestConfig()
-	if s.config == nil {
-		c.Skip("Unable to retrieve test config, skipping docker/plugin restart tests")
+	if s.config == nil || len(s.config.Datastores) < 2 {
+		log.Printf("Restart tests setup skipped - missing or incomplete config")
+		c.Skip("Restart tests setup skipped - missing or incomplete config")
 	}
 
-	s.volumeName = inputparams.GetUniqueVolumeName("restart_test")
-	s.containerName = inputparams.GetUniqueContainerName("restart_test")
+	adminutil.ConfigRemove(s.config.EsxHost)
```

Comment:
this is redundant as per https://github.com/vmware/docker-volume-vsphere/blob/master/tests/utils/admincli/vmgroupmgmt.go#L88